docs/UserGuide.md

```diff
@@ -310,59 +324,96 @@ Example command usages from the current student list shown in Figure 1:
 * `find_student Lee` returns "John Lee" and "Samuel Lee"
 * `find_student Johnz Lee` returns "John Lee", "Johnz Tan" and "Samuel Lee"
 
+Example 2:
+1. You want to find students whose name contains alex, david and roy.
+
+2. Type `find_student alex david roy` and press <kbd>enter</kbd>.
+
+![find_student_example](images/enhao/find_student%20-%20edited.png)
+
+3. The student and session list will be filtered and only contains students and sessions of students whose name contains the keywords specified.
+
+![find_student_result_example](images/enhao/find_student_result%20-%20edited.png)
+
+
 ##### Editing a student: `edit_student`
 
 Edits an existing student's detail(s) in TutorBuddy.
 
 Format: `edit_student STUDENT_INDEX [n/NAME] [p/STUDENT_PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [l/STUDY_LEVEL] [g/GUARDIAN_PHONE_NUMBER] [r/RELATIONSHIP_WITH_GUARDIAN]` <br>
 * `STUDENT_PHONE_NUMBER` and `GUARDIAN_PHONE_NUMBER` should be in Singapore's phone formatting (i.e. starting with either 6, 8 or 9 with 8 digits)
 
-Examples:
-* `edit_student 1 p/98825432 e/alexy@example.com a/Alex street, Block 123, #01-01` edits the student at index 1's `STUDENT_PHONE_NUMBER`, `EMAIL`, and `ADDRESS`
+Example:
+1. You would like to edit `Alex Yeoh` student's phone number, email address and address as they are wrong. 
+2. Type `edit_student 1 p/98825432 e/alexy@example.com a/Alex street, Block 123, #01-01` and press <kbd>enter</kbd>. (This edits the student `Alex Yeoh`'s `STUDENT_PHONE_NUMBER`, `EMAIL`, and `ADDRESS` which is at index 1)
+
+![edit_student_example](images/enhao/edit_student%20-%20edited.png)
+
+3. The edited student information will be reflected in the student list.
+
+![edit_student_result_example](images/enhao/edit_student_result%20-%20edited.png)
+
+<div markdown="span" class="alert alert-primary">:bulb: Tip:
+Use <code>find_student</code> command to filter the list of students you want.<br>
+The <code>STUDENT_INDEX</code> will be based on the current / filtered list.<br>
+(E.g. If John Doe is the 500th student, instead of <code>edit_student 500 [...]</code>, you can use <code>find_student John Doe</code> and
+<code>edit_student 1 [...]</code> if the John Doe you would like to edit is the 1st in the list)
+</div>
+
 
 ##### Deleting a student: `delete_student`
 
 Deletes the specified student from TutorBuddy.
 
 Format: `delete_student STUDENT_INDEX` <br>
 
+Example:
+1. You would like to delete the 2nd student based on the current list of students. 
+2. Type `delete_student 2` and press <kbd>enter</kbd>.
+
+![delete_student_example](images/enhao/delete_student%20-%20edited.png)
+
+3. The 2nd student in the list (`Bernice Yu`) and her associated sessions will be deleted.
```

Comment:
Maybe we shouldn't use the code snippet format cause its not a command or response, perhaps bolding would work?

Reply:
Ah right! Agreed! Have updated in the latest commit.